api/metaschemas/views.py

```diff
@@ -50,11 +51,17 @@ class MetaSchemasList(JSONAPIBaseView, generics.ListAPIView):
     view_category = 'metaschemas'
     view_name = 'metaschema-list'
 
+    # implement ODMFilterMixin
+    def get_default_odm_query(self):
+        return (
+            Q('name', 'in', ACTIVE_META_SCHEMAS) &
+            Q('schema_version', 'eq', LATEST_SCHEMA_VERSION) |
+            Q('name', 'eq', 'Preprint Provider')
```

Comment:
The first two lines query for MetaSchemas that are in the [`ACTIVE_META_SCHEMAS`](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/project/metadata/schemas.py#L41) list **AND** that are [version 2](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/project/metadata/schemas.py#L4). Essentially, it just excludes all of the old v1 registration schemas, and any registration schemas that aren't active yet (which I think is just the [EGAP Project](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/project/metadata/egap-project-2.json)). 

Since the Preprint Provider metaschema is version 1, it would be left out, hence the additional `OR name="Preprint Provider"`. This could also be `OR category="preprintprovider"`.